package.json

```diff
@@ -9,9 +9,10 @@
     "preinstall": "npx only-allow pnpm",
     "wipe": "npkill",
     "wipe:dist": "pnpm -r exec rm -rf ./dist",
-    "build": "pnpm -r --filter !playground build",
-    "build:types": "pnpm -r --filter !playground exec -- tsup --dts-only",
-    "dev": "pnpm -r --parallel --filter !playground dev",
+    "build": "pnpm -r --filter !playground --filter !{examples/*} build",
+    "build:examples": "pnpm -r --filter {examples/*} build",
+    "build:types": "pnpm -r --filter !playground --filter !{examples/*} exec -- tsup --dts-only",
+    "dev": "pnpm -r --parallel --filter !playground --filter !{examples/*} dev",
```

Comment:
we should probably move the playground into the examples folder too, then maybe we could do the same basic app for both.  maybe we should create an issue to create make examples in other frameworks too
- solidjs
- vue
- angular
- vanilla

Reply:
good call - we also might be able to do a hardlink on the config file between them so a change to one updates them all

Reply:
tracked here https://github.com/tonyketcham/flatbread/issues/67

Reply:
Done! I'll further the degree of integration in the PR for #67. For now I've just got it checking that it parses the config correctly and builds, as an e2e test for supporting alternative config file extensions.